homeassistant/components/sensibo/binary_sensor.py

```diff
@@ -56,21 +57,21 @@ class SensiboDeviceBinarySensorEntityDescription(
         entity_category=EntityCategory.DIAGNOSTIC,
         name="Alive",
         icon="mdi:wifi",
-        value_fn=lambda data: data.alive,
+        value_fn=lambda data: cast(Optional[bool], data.alive),
```

Comment:
If you implement PEP 0561 you can avoid the casts

https://peps.python.org/pep-0561/